Readability.js

```diff
@@ -112,7 +112,7 @@ Readability.prototype = {
   // All of the regular expressions in use within readability.
   // Defined up here so we don't instantiate them repeatedly in loops.
   REGEXPS: {
-    unlikelyCandidates: /banner|combx|comment|community|cover-wrap|disqus|extra|foot|header|legends|menu|modal|related|remark|rss|shoutbox|sidebar|skyscraper|sponsor|ad-break|agegate|pagination|pager|popup|yom-remote/i,
+    unlikelyCandidates: /banner|combx|comment|community|cover-wrap|disqus|extra|foot|header|legends|menu|modal|related|remark|replies|rss|shoutbox|sidebar|skyscraper|sponsor|ad-break|agegate|pagination|pager|popup|yom-remote/i,
```

Comment:
Add `replies` to avoid getting wrong top candidate the and fix the failure of [lifehacker-post-comment-load][lifehacker-post-comment-load] test.

[lifehacker-post-comment-load]: https://github.com/mozilla/readability/blob/master/test/test-pages/lifehacker-post-comment-load/source.html#L1202